operator/src/main/resources/scripts/startServer.sh

```diff
@@ -281,10 +285,13 @@ createFolder ${DOMAIN_HOME}/servers/${SERVER_NAME}/security
 copyIfChanged /weblogic-operator/introspector/boot.properties \
               ${DOMAIN_HOME}/servers/${SERVER_NAME}/security/boot.properties
 
-copySitCfgWhileBooting /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig             'Sit-Cfg-CFG--'
-copySitCfgWhileBooting /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/jms         'Sit-Cfg-JMS--'
-copySitCfgWhileBooting /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/jdbc        'Sit-Cfg-JDBC--'
-copySitCfgWhileBooting /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/diagnostics 'Sit-Cfg-WLDF--'
+if [ ! -f /weblogic-operator/introspector/domainzip.secure ]; then
```

Comment:
This check looks a little obscure/fragile to me. Instead, how about checking the DOMAIN_SOURCE_TYPE env var? (You'll need to make sure that this env var is passed to all servers - the operator might only be passing it to the introspector pod right now...)